bakery/tasks/fetch-book.yml

```diff
@@ -18,5 +18,11 @@ run:
       exec 2> >(tee fetched-book/stderr >&2)
       cd fetched-book
       book_dir="$(cat ../book/name)"
-      mkdir -p "$book_dir"
-      neb get -r -d "$book_dir/raw" "$(cat ../book/server)" "$(cat ../book/collection_id)" "$(cat ../book/version)"
+      mkdir -p "$book_dir" ~/.config/
+      server="$(cat ../book/server)"
+      cat >~/.config/nebuchadnezzar.ini <<EOF
+      [settings]
+      [environ-$server]
+      url = https://$server
+      EOF
+      yes | neb get -r -d "$book_dir/raw" "$(cat ../book/server)" "$(cat ../book/collection_id)" "$(cat ../book/version)"
```

Comment:
I didn't know about `yes` until this. I tried it and laughed as a million 'y's showed up in my terminal :joy:

Reply:
hahahaha it's really useful!
I guess we could've add another option to neb so we don't need to use `yes` :p